pkg/common/apis/response.go

```diff
@@ -183,12 +178,24 @@ func wrapResponse(h interceptor.Handler) interceptor.Handler {
 	}
 }
 
+type Validator interface {
```

Comment:
just use interface from envoyproxy/protoc-gen-validate

Reply:
All of them are defined as text templates. i.e. https://github.com/envoyproxy/protoc-gen-validate/blob/main/templates/go/message.go